apps/src/code-studio/components/SublevelCard.jsx

```diff
@@ -50,7 +51,6 @@ const styles = {
     width: 30
   },
   column: {
-    width: 280,
     marginLeft: MARGIN * 2,
```

Comment:
should the `margin: 10` on line 58 be `margin: MARGIN`?

Reply:
Updated